backend/spec/models/answer_spec.rb

```diff
@@ -25,4 +25,22 @@
       expect(answer).to be_valid
     end
   end
+
+  describe '#minimum_one_option' do
+    it do
```

Comment:
add some description for it

Reply:
Add tests for valid answers, remember to always test both cases

Reply:
I think those tests are already there :) as all the other tests are valid right?